### PR TITLE
Update "Creating a new decoupled CMS Website" docs

### PR DESCRIPTION
**Pull request by MikeAlhayek**

### CODE REVIEW

File: src/docs/guides/decoupled-cms/README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The newly created website should be able to run, and look like this:
 
 ```xml
 <PropertyGroup>
-  <TargetFramework>net6.0</TargetFramework>
+  <TargetFramework>net6.0;net7.0</TargetFramework>
```
Comment (agriffard) [inline src/docs/guides/decoupled-cms/README.md:62]: TargetFrameworks with a `s` may be?

File: src/docs/guides/decoupled-cms/README.md [hunk 2]
```diff
@@ -81,18 +81,27 @@ builder.Services.AddOrchardCms();
 ```
 
 !!! warning "Razor Pages"
-    `AddRazorPages` must not be called directly as `services.AddOrchardCms()` already invokes it internally.
+    `builder.Services.AddRazorPages()` must not be called directly as `builder.Services.AddOrchardCms()` already invokes it internally.
 
 - Edit the `Program.cs` file
-- Remove everything after `app.UseStaticFiles();` and replace it by `app.UseOrchardCore();` like this:
+- Add `app.UseOrchardCore();` 
+- Remove andy of the following lines if exists in your `Program.cs` file:
```
Comment (agriffard) [inline src/docs/guides/decoupled-cms/README.md:88]: Who's `andy`? ;-)
`if it exists`

Comment (MikeAlhayek) [reply src/docs/guides/decoupled-cms/README.md]: any, clean that up :)

File: src/docs/guides/decoupled-cms/README.md [hunk 3]
```diff
@@ -81,18 +81,27 @@ builder.Services.AddOrchardCms();
...
 - Edit the `Program.cs` file
-- Remove everything after `app.UseStaticFiles();` and replace it by `app.UseOrchardCore();` like this:
+- Add `app.UseOrchardCore();` 
+- If any of the following lines exists in your `Program.cs` file, remove them:
 
 ```csharp
-   ...
-   
-   app.UseHttpsRedirection();
-   app.UseStaticFiles();
```
Comment (agriffard) [inline src/docs/guides/decoupled-cms/README.md:93]: It still may need this, right?
`app.UseStaticFiles();`

Comment (MikeAlhayek) [reply src/docs/guides/decoupled-cms/README.md:93]: @agriffard 
Yes `app.UseStaticFiles()` is still used. I don't know this isn't part of the **Static File Provider Feature**.

File: src/docs/guides/decoupled-cms/README.md [hunk 4]
```diff
@@ -59,7 +59,7 @@ The newly created website should be able to run, and look like this:
 
 ```xml
 <PropertyGroup>
-  <TargetFramework>net6.0</TargetFramework>
+  <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
```
Comment (sebastienros) [inline src/docs/guides/decoupled-cms/README.md:62]: This doesn't have to be, updating to net7.0 is fine, there is no reason to have two, even more in a sample.

Comment (MikeAlhayek) [reply src/docs/guides/decoupled-cms/README.md:62]: rolled it back

**Pull request merged**